src/lib.rs

```diff
@@ -277,30 +277,41 @@ pub struct Metadata {
     pub max_depth: usize,
 }
 
+/// Helper trait for `core::iter::Peekable`.
+pub trait Peekable: core::iter::Iterator {
+    fn peek(&mut self) -> core::option::Option<&Self::Item>;
+}
+
+impl<I: core::iter::Iterator> Peekable for core::iter::Peekable<I> {
+    fn peek(&mut self) -> core::option::Option<&Self::Item> {
+        core::iter::Peekable::peek(self)
+    }
+}
+
 /// Derive-able trait for structures that can be mutated using serialized paths and values.
 pub trait Miniconf {
     /// Update an element by path.
     ///
     /// # Args
-    /// * `path` - The path to the element.
+    /// * `path` - The path to the element. With '/' as separator.
     /// * `data` - The serialized data making up the content.
     ///
     /// # Returns
     /// The result of the configuration operation.
```

Comment:
Should update docs to indicate it returns the number of bytes used for deserialization